tests/agent/actions/test_generate_data_quality_check_expectations_action.py

```diff
@@ -291,9 +295,6 @@ def test_missing_table_columns_metric_raises_runtime_error(
     mock_data_asset.id = uuid.uuid4()
     mock_data_asset.test_connection.return_value = None
 
-    # Mock the context to return our mock data asset
-    mock_context.data_sources.get.return_value.get_asset.return_value = mock_data_asset
-
```

Comment:
Chaining `return_value` doesn't seem to work like this, and removing this line made no difference regardless.